notebooks/MAST/downloading_transiting_planet_data/downloading_transiting_planet_data.ipynb

```diff
@@ -0,0 +1,961 @@
+{
```

Comment:
<p>why the -1 index? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>